.lycheeignore

```diff
@@ -0,0 +1 @@
+https://sigs.centos.org/virt/tdx/
```

Comment:
Sorry I commented on this on your other PR not here...

Is this Lychee tripping on this link because it works for me.

Reply:
It was weirdly causing an issue earlier, so I followed Steve's suggestion to use lychee.

Reply:
we could try updating to `lycheeverse/lychee-action@v2` and see if the error is fixed.

Reply:
Let's do this in a different PR.